web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -1031,6 +1035,9 @@ fixmystreet.update_pin = function(lonlat, savePushState) {
             category_select.change();
         }
         fixmystreet.run(fixmystreet.set_up.category_groups);
+        if (shadow_category !== old_category) {
```

Comment:
In fact, as this `change()` will go on to call the sub cat change and then the hidden select change, you could probably streamline this a little - if I've understood it right, you only need to do this if shadow_category is something but category_select is not (because if that was something, it would already have therefore set the group inside `set_up.category_groups`).